src/lib/engine/graph.js

```diff
@@ -18,7 +18,10 @@ export function relationshipType(edges) {
 }
 
 export function create(graphAttributes) {
-  return _.pick(graphAttributes, ['metrics', 'guesstimates', 'simulations'])
+  return {
+    ..._.pick(graphAttributes, ['metrics', 'guesstimates', 'simulations']),
+    ..._.pick(graphAttributes.factBank, ['globals', 'organizationFacts']),
```

Comment:
can this just be called ``graphAttributes.facts``?  

I do like the names 'globals' and 'organizationFacts"